maas-schemas-ts/jestconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "globals": {
```

Comment:
Would be great if jest configuration would not require specifying it command line, i.e. add it to `package.json` or rename file to `jest.config.js`